azure-pipelines/builds/ci.yml

```diff
@@ -15,24 +15,69 @@ variables:
   - name: Codeql.Enabled
     value: true
 
+  - name: defaultPoolName
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      value: $[replace(replace(eq(contains(coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranch'], 'refs/heads/main'), 'release'), 'true'), True, 'NetCore-Svc-Public' ), False, 'NetCore-Public')]
+    ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+      value: $[replace(replace(eq(contains(coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranch'], 'refs/heads/main'), 'release'), 'true'), True, 'NetCore1ESPool-Svc-Internal'), False, 'NetCore1ESPool-Internal')]
+
+  - name: defaultPoolDemandsLinux
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      value: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
+    ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+      value: ImageOverride -equals Build.Ubuntu.1804.Amd64
+
+  - name: defaultPoolDemandsWindows
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      value: ImageOverride -equals 1es-windows-2022-open
+    ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+      value: ImageOverride -equals 1es-windows-2022
```

Comment:
These variables are only used in a single place. It would be less verbose to inline them and not define a separate variable for them. See https://github.com/dotnet/aspire/blob/74c40701ec6385bcccc2452863ffb3585d36601e/eng/pipelines/azure-pipelines.yml#L103-L108 for an example.

Reply:
(I am unable to reopen this merged PR. I have linked a [new PR with these changes](https://github.com/dotnet/source-build-reference-packages/pull/830/files#diff-33ae5736679b1efe12fde9f3fb58dbe360dd8a8bb32e5b98620bf7d644114f19R26).)

This makes much more sense. Thank you for pointing this out. I have addressed this comment here: https://github.com/dotnet/source-build-reference-packages/pull/830/files#diff-33ae5736679b1efe12fde9f3fb58dbe360dd8a8bb32e5b98620bf7d644114f19R26